### PR TITLE
Fix minor issues in http cache

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | n/a |
| Versions      | 3.3 and up |
| Edition       | all |

Fix minor issues in http cache

#### Checklist

- [ ] Text renders correctly

### CODE REVIEW

**Comment by dabrt** [comment]
Hi @vidarl , in content_aware_cache.md, example esi_include values contain references to eZ Systems and eX Platform. Should it stay like that? If not please suggest what we should put in there.

**Comment by dabrt** [comment]
> Hi @vidarl , in content_aware_cache.md, example esi_include values contain references to eZ Systems and eZ Platform. Also a number of references to eZ Systems survived in `sub ez_...` and `call ez_...` calls.
> 
> Should it stay like that? If not please suggest what we should put in there.

@vidarl This issue has also been identified in a parallel PR here:
https://github.com/ibexa/http-cache/pull/46#pullrequestreview-1968496466

**Review by vidarl — APPROVED** [review]

File: docs/infrastructure_and_maintenance/cache/http_cache/context_aware_cache.md [hunk 1]
```diff
@@ -204,6 +204,6 @@ public function addPreferenceHash(FilterResponseEvent $event)
 ```php
 $response->setVary('X-User-Preference-Hash');
 
-// If you _also_ need to vary on eZ permissions, instead use:
+// If you _also_ need to vary on Ibexa Platform permissions, instead use:
```
Comment (mnocon) [inline docs/infrastructure_and_maintenance/cache/http_cache/context_aware_cache.md:207]:
```suggestion
// If you _also_ need to vary on [[= product_name =]] permissions, instead use:
```
I think we prefer the name `Ibexa DXP`?

File: docs/infrastructure_and_maintenance/cache/persistence_cache.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ For further details on which calls are cached or not, see details in the [Symfon
 which has info on cache use in two places:
 
 - Symfony Cache tab: for Symfony Cache itself, the tab shows cache lookups to cache backends
-- eZ Platform tab: shows calls made to database back end, and if they are cached or not
+- Ibexa tab: shows calls made to database back end, and if they are cached or not
```
Comment (mnocon) [inline docs/infrastructure_and_maintenance/cache/persistence_cache.md:43]:
```suggestion
- [[= product_name_base =]] tab: shows calls made to database back end, and if they are cached or not
```
I think vale is right here - if we rename the company we will probably rename this tab as well 😅

**Review by mnocon — APPROVED** [review]
Looks good to me, just two small suggestions

**Pull request merged**